micro-benchmarks/DRB189-bar3_yes.c

```diff
@@ -0,0 +1,119 @@
+/*
+!!!~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~!!!
+!!! Copyright (c) 2017-20, Lawrence Livermore National Security, LLC
+!!! and DataRaceBench project contributors. See the DataRaceBench/COPYRIGHT file for details.
+!!!
+!!! SPDX-License-Identifier: (BSD-3-Clause)
+!!!~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~!!!
+ */
+
+/*
+ * This is a program based on a dataset contributed by
+ * Wenhao Wu and Stephen F. Siegel @Univ. of Delaware.
+
+ * one synchronization commented out.
+ * enters.  So race on x can occur.
+ * Data race pair: x@39:7:W vs. x@51:7:W
```

Comment:
the line numbers do not point to uses of variable x. @LChenGit

Reply:
it is changed to: "Data race pair: x@104:9:W vs. x@111:9:W"